packages/x-date-pickers/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -198,14 +190,6 @@ export const DateTimePickerToolbar = <TDate extends unknown>(
           </DateTimePickerToolbarTimeContainer>
         </DateTimePickerToolbarRoot>
       )}
-      {showTabs && (
-        <DateTimePickerTabs
-          dateRangeIcon={dateRangeIcon}
-          timeIcon={timeIcon}
-          view={openView}
-          onChange={setOpenView}
-        />
-      )}
     </React.Fragment>
```

Comment:
Seems that `React.Fragment` is not used anymore.

What about removing this `wrapperVariant !== 'desktop'` on top and use it as a default value for `showToolbar`

Reply:
Related to #5841

Reply:
Thanks for pointing this out. 🙇 
By the way, we already default to your mentioned condition via 
```js
const toShowToolbar = showToolbar ?? wrapperVariant !== 'desktop'
```
So, removing this hard-coded logic in Toolbar will easily solve 5841 🎉 👍

Reply:
😂
- issue solved: 1
- line of code added: -1